tests/functional/test_api.py

```diff
@@ -7,19 +7,23 @@
 from ghmirror.app import APP
 from ghmirror.data_structures.monostate import UsersCache
 from ghmirror.data_structures.monostate import GithubStatus
-from ghmirror.core.constants import REQUESTS_TIMEOUT
+from ghmirror.core.constants import REQUESTS_TIMEOUT, PER_PAGE_ELEMENTS
 from ghmirror.utils.wait import wait_for
 
 
 class MockResponse:
-    def __init__(self, content, headers, status_code, user=None):
+    def __init__(self, content, headers, status_code, user=None, links={}, json_content=None):
```

Comment:
`links={}` is unsafe (we should probably lint the tests, pylint would tell you so)

Reply:
Sure, sounds like a plan!